docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -80,6 +80,24 @@ \section{Coding Standard}
 
 \section{Project Scheduling}
 
-\wss{How will the project be scheduled?}
+The general scheduling of the project will follow a weekly sprint format. At the beginning
+of each week, a plan will be made based on the deliverables due and progress made from previous sprints. The
+work will then be distributed afterwards such that all members have tasks they have
+to work on. This structure will be done to ensure feasibility regarding weekly
```

Comment:
"This structure will be done to ensure feasibility regarding weekly"

I feel like nitpicking, but the correct way to phrase the sentence would be:

"This structure will ensure feasibility regarding weekly ..."